src/components/Book/BookAbout/BookAbout.tsx

```diff
@@ -1,21 +1,56 @@
 import React, { useEffect, useState } from 'react';
 import classes from './BookAbout.module.scss';
 import image from 'util/327752.jpeg';
-import { useParams } from 'react-router-dom';
+import { Link, useParams } from 'react-router-dom';
 import Book from 'model/Book';
 import axios from '../../../axios/axiosConfig';
 import Button from 'components/core/Button/Button';
+import authService from 'services/authService';
+import Input from 'components/core/Input/Input';
+
+interface BookCopy {
+  id: null;
+  identifier: string;
+  bookId: string | undefined;
+}
 
 const BookAbout: React.FC = () => {
   const { id } = useParams();
   const [book, setBook] = useState<Book>({} as Book);
+  const [days, setDays] = useState<string>('');
+
+  const { isAdministrator } = authService;
 
   const getBook = () => {
     axios.get(`/book/${id}`).then((res) => {
       setBook(res.data);
     });
   };
 
+  const rentBook = () => {
+    axios
+      .post(`/rent/book/${id}/user?rentPeriod=${days}`)
+      .then((res) => {
+        console.log(res);
+      })
+      .catch((err) => console.log(err.response.data.message));
+  };
+
+  const createBookCopy = () => {
+    const data: BookCopy = { id: null, identifier: crypto.randomUUID(), bookId: id };
+
+    axios
+      .post(`/book/${id}/book-copy`, data)
+      .then((res) => {
+        console.log(res);
+      })
+      .catch((err) => console.log(err.response.data.message));
+  };
+
+  const changeDaysHandler = (value: string) => {
+    setDays(value);
+  };
+
```

Comment:
API calls should be separated from Component, inside services ,because we want to Component looks like [Pure function](https://en.wikipedia.org/wiki/Pure_function)